roles/container-engine/containerd/templates/hosts.toml.j2

```diff
@@ -5,4 +5,7 @@ server = "https://{{ item.prefix }}"
 {% if mirror.skip_verify is defined %}
   skip_verify = {{ mirror.skip_verify | default('false') | string | lower }}
 {% endif %}
+{% if mirror.override_path is defined %}
+  override_path = {{ mirror.override_path | default('false') | string | lower }}
+{% endif %}
```

Comment:
I'd drop the if. 
```suggestion
  override_path = {{ mirror.override_path | default('false') | string | lower }}
```
false is containerd default, so that's consistent.


(In fact I'd drop the if on skip_verify as well ^)

Reply:
Thanks @VannTen 

After thinking about the idea twice, there is a problem. 

If "the `if` be dropped", every `host.toml` file will contain 
```
skip_verify = false
override_path = false
```
although the `skip_verify` and `override_path` is not defined.
It's not necessary, and it makes the config file simpler.

So, I think the original PR is reasonable, and it keeps the same with `skip_verify`, making the code easier to read. :-)